modules/post/windows/gather/file_in_raw_ntfs.rb

```diff
@@ -0,0 +1,79 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class Metasploit3 < Msf::Post
+  include Msf::Post::Windows::Priv
+  require "rex/parser/fs/ntfs"
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'         => 'Windows File Gathering In Raw NTFS',
+      'Description'  => %q{
+          This module gather file using the raw NTFS device, bypassing some Windows restriction.
+          Gather file from disk bypassing restriction like already open file with write right lock.
+          Can be used to retreive file like NTDS.DIT
+      },
+      'License'      => 'MSF_LICENSE',
+      'Platform'     => ['win'],
+      'SessionTypes' => ['meterpreter'],
+      'Author'       => ['Danil Bazin <danil.bazin[at]hsc.fr> @danilbaz'],
+      'References'   => [
+        [ 'URL', 'http://www.amazon.com/System-Forensic-Analysis-Brian-Carrier/dp/0321268172/' ]
+      ]
+    ))
+    register_options(
+      [
+        OptString.new('FILE', [true, 'The FILE to retreive from the Volume raw device', ""])
```

Comment:
Use `nil` instead of the `""` empty string should cause the validator to error if the user hasn't supplied a path.